### PR TITLE
Integrate htmlTransform function from polymer-build.

### PR DESCRIPTION
**Pull request by aomarks**
This will fail on Travis until https://github.com/Polymer/polymer-build/pull/341 is released.

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- ## Unreleased -->
```
Comment (rictic) [inline CHANGELOG.md]: Uncomment

File: package.json [hunk 2]
```diff
@@ -53,9 +52,8 @@
     "mime": "^1.3.4",
     "mz": "^2.4.0",
     "opn": "^3.0.2",
-    "parse5": "^2.2.3",
     "pem": "^1.8.3",
-    "polymer-build": "^3.0.0-pre.2",
+    "polymer-build": "^3.0.0-pre.4",
```
Comment (rictic) [inline package.json]: `=` instead of `^`?

File: CHANGELOG.md (hunk 1 above)
Comment (aomarks) [reply CHANGELOG.md]: Done.

File: package.json (hunk 2 above)
Comment (aomarks) [reply package.json]: Done.

**Pull request merged**